### ISSUE TITLE
SOLR failing to start 

### ISSUE THREAD

**Issue by stuartcam** [Sitecore/docker-images#312]
Hi guys,

I'm trying to use the latest images on AWS (using  -IsolationModeBehaviour EngineDefault rather than hyperv isolation) and everything seems to work fine until I try and start up an XP configuration 

```
PS C:\sitecore\docker-images\windows\tests\9.3.x> docker-compose --file .\docker-compose.xp.yml up
Starting 93x_solr_1 ... done
Creating 93x_sql_1  ... done
Creating 93x_xconnect-indexworker_1 ... done

ERROR: for xconnect  Container "b9e3c9703b95" is unhealthy.
ERROR: Encountered errors while bringing up the project.
```

If I just try to bring up solr on it's own and no other services I get the following (the logs for xconnect say it can't find solr) 

```
Attaching to 93x_solr_1
solr_1  | SOLR_PATH=c:\\solr
solr_1  | SOLR_PORT=8983
solr_1  | INSTALL_PATH=c:\\clean
solr_1  | DATA_PATH=c:\\data
solr_1  | "### Existing Sitecore solr cores found in 'c:\\data'..."
solr_1  | "### Preparing Solr cores..."
solr_1  | Could Not Find c:\data\write.lock
solr_1  | "### Starting Solr..."
solr_1  |
solr_1  | ERROR: Java 1.8 or later is required to run Solr. Current Java version is:  (detected major: 0)
solr_1  |
93x_solr_1 exited with code 1
```

Not sure really where to go from here? Sitecore XM starts up just fine (albeit with the solr cores missing in control panel so solr clearly isnt' working there either)

Additionally, do I have to edit my hosts file for solr and xconnect dns to resolve or is this automatic? Outside of the solr java error it seems to not be able to find the dns for them (or is it just a by product of it not starting correctly)

**Comment by stuartcam** [comment]
From what I can tell the dockfile for sitecore-openjdk is pointing to a version of openjdk that no longer exists - is it as straight forward as updating that?

**Comment by stuartcam** [comment]
Manually edited the file to point to the latest version and it seems to work now, created a pr

**Comment by pbering** [comment]
Closed by #313
